aerospike_helpers/operations/map_operations.py

```diff
@@ -48,11 +48,12 @@
 def map_set_policy(bin_name: str, policy, ctx: Optional[list] = None):
     """Creates a map_set_policy_operation.
 
-    The operation allows a user to set the policy for the map.
+    Server sets map policy attributes. Server does not return a value.
```

Comment:
```suggestion
    This operation sets map policy attributes server-side for the designated bin. Server does not return a value.
```

Reply:
I think the server is setting the attributes for a map, not a bin.